app/stories/modal.stories.jsx

```diff
@@ -362,27 +363,89 @@ story('Marketplace/GetDataTokensDialog')
         />
     ))
 
+story('Marketplace/GetDataTokensDialog(iPhone)')
+    .add('default', () => (
+        <GetDataTokensDialog
+            onCancel={action('onCancel')}
+        />
+    ), {
+        viewport: {
+            defaultViewport: 'iPhone',
+        },
+    })
+
 story('Marketplace/GetCryptoDialog')
     .add('default', () => (
         <GetCryptoDialog
             onCancel={action('onCancel')}
         />
     ))
 
+story('Marketplace/GetCryptoDialog(iPhone)')
```

Comment:
I suggest adding them like this for better grouping:
```js
story('Marketplace/GetCryptoDialog')
     .add('default', () => (
         <GetCryptoDialog
             onCancel={action('onCancel')}
         />
     ))
     .add('default (iPhone)', () => (
         <GetCryptoDialog
             onCancel={action('onCancel')}
         />
     ), {
         viewport: {
             defaultViewport: 'iPhone',
         },
     })
```

Reply:
Ah, much better. Grouped and a green build. I will merge this now.